assets/src/features/leaving-page/utils/sendForm.tsx

```diff
@@ -0,0 +1,21 @@
+import { Inputs } from "../components/Questionnaire";
+import { BASE_URL } from "./formInfo";
+import parseToQuery from "./parseToQuery";
+
+const sendForm = async (form: Inputs) => {
+  const query = parseToQuery(form);
+  const url = BASE_URL.concat(query);
+
+  // We use google url with url embedded answers.
+  // A simple fetch is sufficient for our needs
+  // The {mode: "no-cors"} allows the url to be fetched.
+
+  // Since the request is opaque, we have no direct information
+  // if the response has completed successfully or not.
+  // In case of wrong input values, the `fetch` function logs `REQ_ERR` on stderr.
+  fetch(url, { mode: "no-cors" }).then((response) => {
+    console.log("Response", response);
+  });
```

Comment:
since it's async function you should rather use `await`
```suggestion
  const response = await fetch(url, { mode: "no-cors" });
  console.log("Response", response);
```

Reply:
Probably the `console.log` should be thrown out regardless of using `then` or `await`.